tests/performance/apdex-score/test/page-objects/page.js

```diff
@@ -176,86 +176,31 @@ module.exports = class Page {
     await this.assertMany(form.postSubmitAssert);
   }
 
-  // ToDo: clean all these below after settings are done
-
-  get btnCustom() {
-    return $('//*[@text="Custom"]');
-  }
-
-  get tabMessages() {
-    return $('//*[@text="Messages"]');
-  }
-
-  get tabTasks() {
-    return $('//*[@text="Tasks"]');
-  }
-
-  get tabReports() {
-    return $('//*[@text="Reports"]');
-  }
-
-  get tabPeople() {
-    return $('//*[@text="People"]');
-  }
-
-  get tabPerformance() {
-    return $('//*[@text="Performance"]');
-  }
-
-  get tabVHTSummary() {
-    return $('//*[contains(@text, "Summary")]');
-  }
-
-  get tabAnalytics() {
-    return $('//*[contains(@text, "Analytics")]');
-  }
-
-  get tabDropdown() {
-    return $('//android.view.View[@text=""]');
-  }
-
-  get menuItemSyncNow() {
-    return $('//android.view.MenuItem[@text="Sync now"]');
-  }
-
-  get menuTextSyncStatus() {
-    return $('(//*[@resource-id="header-dropdown"]//android.view.View)[2]');
-  }
-
-  get menuTextSyncTime() {
-    return $('//android.view.View[contains(@text, "Last sync")]');
-  }
-
-  get menuItemAbout() {
-    return $('//android.view.MenuItem[@text="About"]');
-  }
-
-  get menuItemSettings() {
-    return $('//android.view.MenuItem[@text="User settings"]');
-  }
+  async fillUpFormOnly(form, commonElements) {
```

Comment:
You're adding this new function because tasks don't come from a FAB button, right?

In that case, to avoid repeating code add a new setting option in the forms like:
```js
 "forms": {
     "patientTask": {
           "navigation": [.......],
           "skipFAB": true, <-----This 
           "pages": [.......]
```
Then update the README.md in [the form's section to describe that new option ](https://github.com/medic/cht-core/tree/complete-apdex-automation-tasks/tests/performance/apdex-score#forms)

Lastly, go the [fillUpForm function](https://github.com/medic/cht-core/blob/cb9eb8b2072f73e382dd11b7ebf6dddf5b50f158/tests/performance/apdex-score/test/page-objects/page.js#L142) and add a IF statement for that

```js
 async fillUpForm(form, commonElements){
    const FAB_SELECTOR = commonElements?.fab || '//android.widget.Button[not(@text="Actions menu")]';
    const FAB_LIST_TITLE = commonElements?.fabListTitle || '//android.widget.TextView[@text="New"]';
    const FORM_SUBMIT_SELECTOR = commonElements?.formSubmit || '//android.widget.Button[@text="Submit"]';
    const FORM_PAGE_NEXT_SELECTOR = commonElements?.formNext || '//android.widget.Button[@text="Next >"]';

   if (form.skipFAB) {
      await this.clickElement(FAB_SELECTOR);
      await this.waitForDisplayedAndRetry(FAB_LIST_TITLE);
   } 
    await this.navigate(form.navigation);